nrt/data/DynaFlow/cases.py

```diff
@@ -107,3 +107,13 @@
 job_file = os.path.join(os.path.dirname(__file__), "HvdcPowerTransfer", "HvdcPowerTransfer.jobs")
 
 test_cases.append((case_name, case_description, job_file, 1, standardReturnCodeType, standardReturnCode))
+
+####################################################################################
+#  Small System with with a Load Restoration following a line disconnection #
```

Comment:
Maybe we can precise somewhere that we use the CPP model ? Just a proposal.

Reply:
ok, I can add it in the description